mathics/builtin/datentime.py

```diff
@@ -1067,10 +1084,10 @@ class Now(Predefined):
     summary_text = "current date and time"
 
     def evaluate(self, evaluation):
-        return Expression("DateObject").evaluate(evaluation)
+        return Expression(SymbolDateObject.evaluate(evaluation))
 
 
-if sys.platform != "win32" and ("Pyston" not in sys.version):
+if sys.platform != "win32" and not hasattr(sys, "pyston_version_info"):
```

Comment:
The preferred way for checking for Pyston is like this. See https://github.com/pyston/pyston#checking-for-pyston-at-runtime